.github/workflows/ci.yaml

```diff
@@ -66,7 +66,7 @@ jobs:
 
     - uses: peaceiris/actions-gh-pages@v3
       name: Publish Documentation
-      if: github.event_name == 'push'
+      # if: github.event_name == 'push'
```

Comment:
I think we should keep this `if` and only publish documentation of the `master` branch. WDYT?

Reply:
Yes we should. And same for docker images if we don't already.

Reply:
Yes of course: I commented it out because I wanted to check how the haddock renders online.